sky/spot/spot_utils.py

```diff
@@ -199,22 +200,26 @@ def cancel_job_by_name(job_name: str) -> str:
 def stream_logs_by_id(job_id: int, follow: bool = True) -> str:
     """Stream logs by job id."""
     controller_status = job_lib.get_status(job_id)
-    while (controller_status != job_lib.JobStatus.RUNNING and
-           (controller_status is None or not controller_status.is_terminal())):
-        status_str = 'None'
-        if controller_status is not None:
-            status_str = controller_status.value
-        logger.info(
-            'Waiting for the spot controller process to be RUNNING (status: '
-            f'{status_str}).')
-        time.sleep(_LOG_STREAM_CHECK_CONTROLLER_GAP_SECONDS)
-        controller_status = job_lib.get_status(job_id)
-
-    job_status = spot_state.get_status(job_id)
-    while job_status is None:
-        logger.info('Waiting for the spot job to be started.')
-        time.sleep(1)
+    status_msg = ('[bold cyan]Waiting for controller, controller status: {}.[/]'
```

Comment:
```suggestion
    status_msg = ('[bold cyan]Waiting for controller process to start/to be RUNNING (status: {}).[/]'
```
Reasons:
- distinguish "controller" and "controller process" 
  - We may want to avoid this confusion by renaming "controller process" to "monitor process" (for future)
- "waiting for X" is unclear to me

Reply:
Done. Thanks!